### PR TITLE
November 2024 news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+We discussed the [GitHub workflow](/blog/2023/09/26/how-to-translate-a-hugo-blog-post-with-babeldown/) and the [babelquarto](https://docs.ropensci.org/babelquarto/) and [babeldown](https://docs.ropensci.org/babeldown/) packages. 
+We also reviewed the [translation guidelines](https://translationguide.ropensci.org/) and started working on a Portuguese glossary for the translations.
+
+Participants worked reviewing the [dev guide](https://devdevguide.netlify.app/pt/index.pt.html). 
+As a result, we now have two more chapters with their first review started, two chapters with the second review under work, and five chapters merged with the complete translation. We also started the translation of our Code of Conduct.  🚀 
+
+We are very grateful to Pedro Faria, Beatriz Milz, Francesca Palmeira, Rafael Fontenelle, Ildeberto Vasconcelos,  Samuel Carleial, and Ariana Cabral for their contribution during the Traslatón 🙏.
+
+ If you want to participate in this collaborative translation effort, please [check our project](https://github.com/orgs/ropensci/projects/7/views/1) and add a comment to the pull request you want to review.
+
```
Comment (yabellini) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:71]:
```suggestion
 
 ## Community Call "A comunidade R fala português" Resources
 
 All the [resources for our first community call in português]( /commcalls/translation-portuguese/) are now available on our website.
 
 Check the video, speakers slides, and links to other resources related to translation efforts in the R Community.
  
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,645 @@
...
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### A fast R-universe!
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:46]:
```suggestion
### A fast R-Universe!
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+
+###  Tradução + Hackathon = Traslatón rOpenSci
+
+We had a very nice and productive event during [LatinR](/events/latinr-2024/), with people registered from thirteen countries. 
+Some people stayed for the whole event and others participated during some of the working sessions.
+ 
+We discussed the [GitHub workflow](/blog/2023/09/26/how-to-translate-a-hugo-blog-post-with-babeldown/) and the [babelquarto](https://docs.ropensci.org/babelquarto/) and [babeldown](https://docs.ropensci.org/babeldown/) packages. 
+We also reviewed the [translation guidelines](https://translationguide.ropensci.org/) and started working on a Portuguese glossary for the translations.
+
+Participants worked reviewing the [dev guide](https://devdevguide.netlify.app/pt/index.pt.html).
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:65]:
```suggestion
Participants worked on reviewing the rOpenSci [dev guide](https://devdevguide.netlify.app/pt/index.pt.html). 
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,645 @@
...
+We had a very nice and productive event during [LatinR](/events/latinr-2024/), with people registered from thirteen countries. 
+Some people stayed for the whole event and others participated during some of the working sessions.
+ 
+We discussed the [GitHub workflow](/blog/2023/09/26/how-to-translate-a-hugo-blog-post-with-babeldown/) and the [babelquarto](https://docs.ropensci.org/babelquarto/) and [babeldown](https://docs.ropensci.org/babeldown/) packages. 
+We also reviewed the [translation guidelines](https://translationguide.ropensci.org/) and started working on a Portuguese glossary for the translations.
+
+Participants worked reviewing the [dev guide](https://devdevguide.netlify.app/pt/index.pt.html). 
+As a result, we now have two more chapters with their first review started, two chapters with the second review under work, and five chapters merged with the complete translation. We also started the translation of our Code of Conduct.  🚀 
+
+We are very grateful to Pedro Faria, Beatriz Milz, Francesca Palmeira, Rafael Fontenelle, Ildeberto Vasconcelos,  Samuel Carleial, and Ariana Cabral for their contribution during the Traslatón 🙏.
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:68]: Could like to rOpenSci author pages or personal websites?
```suggestion
We are very grateful to Pedro Faria, Beatriz Milz, Francesca Palmeira, Rafael Fontenelle, Ildeberto Vasconcelos,  Samuel Carleial, and Ariana Cabral for their contributions during the Traslatón 🙏.
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,645 @@
...
+
+ If you want to participate in this collaborative translation effort, please [check our project](https://github.com/orgs/ropensci/projects/7/views/1) and add a comment to the pull request you want to review.
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:70]:
```suggestion
 If you want to participate in this collaborative translation effort, please [check out our project](https://github.com/orgs/ropensci/projects/7/views/1) and add a comment to the pull request you would like to review.
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,645 @@
...
+ 
+     
+### Coworking
+
+Read [all about coworking](/blog/2023/06/21/coworking/)!
+
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts.
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:78]: What do you think @maelle? Time to consolidate a little bit?
```suggestion
Join us for [social coworking & office hours](/blog/2023/06/21/coworking/) monthly on first Tuesdays! 
Hosted by Steffi LaZerte and various community hosts. 
```

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,645 @@
...
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, December 3rd, 09:00 Australia Western (01:00 UTC)](/events/coworking-2024-12), "Getting Involved in the Antarctic/Southern Ocean rOpenSci Community" with cohosts [Michael Sumner](/author/michael-sumner ) and [Ben Raymond](/author/ben-raymond), and [Steffi LaZerte](/author/steffi-lazerte).
+    - [Explore rOpenSci R packages](/blog/2018/11/13/antarctic/#demo) supporting Antarctic and Southern Ocean Research.
+    - Chat with our cohosts to find out how you can be involved in this community.
+    
+- [Tuesday, January 14th 14:00 European Central (13:00 UTC)](/events/coworking-2025-01/), "Working with Parquet in R" with cohosts [Rainer M Krug](/author/rainer-m-krug/), and [Steffi LaZerte](/author/steffi-lazerte).
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:87]:
```suggestion
- [Tuesday, January 14th, 14:00 European Central (13:00 UTC)](/events/coworking-2025-01/), "Working with Parquet in R" with cohosts [Rainer M Krug](/author/rainer-m-krug/), and [Steffi LaZerte](/author/steffi-lazerte).
```

File: content/blog/2024-11-29-news-november-2024/index.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+     * [mbquartR](https://github.com/ropensci/software-review/issues/658), Finding Manitoba Quarter Sections. Submitted by [Alex Koiter](http://www.alexkoiter.ca). 
+
+    * [sits](https://github.com/ropensci/software-review/issues/596), Satellite Image Time Series Analysis for Earth Observation Data Cubes. Submitted by [Gilberto Camara](https://www.gilbertocamara.org). 
+
+    * [fwildclusterboot](https://github.com/ropensci/software-review/issues/546), Fast Wild Cluster Bootstrap Inference for Linear Models. Submitted by [Alexander Fischer](https://s3alfisc.github.io/blog/).  (Stats).
+
+* Four at ['2/seeking-reviewer(s)'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A2/seeking-reviewer(s)):
+
+     * [fireexposuR](https://github.com/ropensci/software-review/issues/659), Compute and Visualize Wildfire Exposure. Submitted by [Air Forbes](https://wildfireanalytics.org/forbes.html).
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.md:119]: fireexposuR is listed twice (again at the end)

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, looking good 😎 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'editor' is filled with your name
* [X] YAML field 'preface' is present if necessary

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,645 @@
...
+```
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
```
Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:45]: ### Looking for Maintainers to Support First-Time Contributors

**Now open to non-rOpenSci package maintainers!**

Making your first contribution to Open Source can be both empowering and intimidating. As such, we’re exited to announce a special series of activities to support first-time contributors! 🎉

Are you an package maintainer who would like to help someone make their first contribution? We limited our initial call to rOpenSci package maintainers, but we still have some room, so have decided to open this up to all package maintainers!

See our [blog post](https://ropensci.org/blog/2024/10/22/first-time-contributions/) for more details and how to sign up (by November 24).

Comment (steffilazerte) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:45]: @yabellini, what do you think? December 9th? Or should it be next week, December 2nd?
I will change the blog post with a banner to note the differences, could you update the language of the AirTable?
Thanks!
```suggestion
### Looking for Maintainers to Support First-Time Contributors

**Now open to non-rOpenSci package maintainers!**

Making your first contribution to Open Source can be both empowering and intimidating. As such, we’re exited to announce a special series of activities to support first-time contributors! 🎉

Are you an package maintainer who would like to help someone make their first contribution? We limited our initial call to rOpenSci package maintainers, but we still have some room, so have decided to open this up to all package maintainers!

See our [blog post](https://ropensci.org/blog/2024/10/22/first-time-contributions/) for more details and how to sign up (by December 9th).
```

Comment (yabellini) [reply content/blog/2024-11-29-news-november-2024/index.Rmd:45]: Looks good and I think the new data is good too.

File: content/blog/2024-11-29-news-november-2024/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,661 @@
...
+Don't miss the blog post [Handling CRAN Requirements for Web API R Packages](https://blog.thecoatlessprofessor.com/programming/r/api-packages-and-cran-requirements/) by James Balamuta!
+
+Also relevant, this chapter of the HTTP testing in R book: [CRAN- (and Bioconductor) preparedness for your tests](https://books.ropensci.org/http-testing/cran-preparedness.html).
+
+### Update your GitHub Actions workflow
+
+If you were still using version 3 of the Artifacts action, upgrade as your workflows will start [failing](https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/).
+Thanks to Hugo Gruson for pointing this out!
+
```
Comment (assignUser) [inline content/blog/2024-11-29-news-november-2024/index.Rmd:648]:
````suggestion

Github provides the [dependabot service](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot) that can help you keep actions updated hassle free! Just add the following simple yaml file as `.github/dependabot.yml` in your repo and a friendly bot will open PRs to keep you workflows up to date!
```{verbatim, lang = "yaml"}
version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "monthly"
    open-pull-requests-limit: 10
```
````

**Pull request merged**